### DISCUSSION TITLE
Update frequency Download Server?

### DISCUSSION THREAD

**Discussion by a-einstein** [mapsforge/mapsforge#1301] [Q&A] [answered]
I was very happy to find the [Download Server](https://download.mapsforge.org/) for maps, which enables me to download Openstreetmaps outside of an Android application, on which it would be impossible,  be charged for, or even have low resolution. 
An extra aspect of course is that Openstreetmaps is community based, and I am even contributing to it myself.
I was surprised to be able to load entire (smaller) countries.

But I noticed that the maps may be several months old. Update frequency seems to be be 4 months or more. So currently I cannot even see my own edits. I assume it is not yourself who is generating the maps, but I have some questions anyway.

So my questions are.
- What IS the update frequency?
- Can that be increased? Ideally to like daily?
- Are there other sources like this?

Thanks.

**Comment by applantation** [comment, marked as answer]
Thanks for your interest in the download server.

I try to keep all regions up to date, so that maps remain useful, but i
never intended nor am I am able to to provide an update frequency that is
useful for editing purposes. Even if we had the fastest server this would
be impossible as the data is not sourced directly from OSM servers, but via
Geofabrik.de servers that provide regional cutouts and sea borders - so
everything gets delayed by a few days minimum before we even start.

So my questions are.
>
>    - What IS the update frequency?
>
> There is no fixed update cycle: the biggest problem is that certain
regions take very long to compile into maps and therefore often fail
because of server memory limitations. That makes certain areas way behind
smaller regions.


>    - Can that be increased? Ideally to like daily?
>
>
No, this is with the current infrastructure not possible. Server processing
power is at its limits.

>
>    - Are there other sources like this?
>
> I don't know, but the whole process is open source, so it is possible for
others to do with little effort.

Thanks for the interest,

Ludwig

**Reply by a-einstein** [reply]
Thanks for your effort.

**Comment by devemux86** [comment]
> Are there other sources like this?

Main page mentions the available [Mapsforge map providers](https://github.com/mapsforge/mapsforge/blob/master/docs/Mapsforge-Maps.md).

e.g. [OpenAndroMaps](https://www.openandromaps.org/en)

---

If you are interested in map creation, you can check:
- [Map creation guide](https://github.com/mapsforge/mapsforge/blob/master/docs/MapCreation.md)
- [New automatic generation for Mapsforge maps / pois using Geofabrik](https://github.com/mapsforge/mapsforge-creator)
- [Old automatic generation for Mapsforge maps using planet.osm](https://github.com/mapsforge/mapsforge-mapcreator)

**Reply by a-einstein** [reply]
Thanks, 
I had not noticed that list, or found any other sources yet.
I will have a look at the listed sources.

**Reply by a-einstein** [reply]
I just tried 2 of them.
They work, now it's just a matter of picking the most recent one.
There is also a difference in rendering.
Great!
Only thing to wish for is altitudes and shading, which seems to be a thing.
